build.gradle

```diff
@@ -74,6 +74,7 @@ project(':accurest-core') {
 		compile "com.github.tomakehurst:wiremock:$wiremockVersion"
 		testCompile 'cglib:cglib-nodep:2.2'
 		testCompile 'org.objenesis:objenesis:2.1'
+		testCompile 'com.jayway.restassured:spring-mock-mvc:2.5.0'
```

Comment:
So that GroovyShell (the changed snippet below) work without throwing compilation errors at importing RestAssuredMockMvc.